backend/game/turn_actions/test_heal.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+import game.values.constants as constants
+import game.turn_actions.heal as heal
+from game.player.player import Player
+
+
+@pytest.fixture(autouse=True)
+def player():
+    player: Player = Player()
+    return player
+
+
+def test_heal_only_in_tokyo(player):
+    player.leave_tokyo()
+    starting_health = player.current_health
+    player.update_health_by(-3)
+    heal.heal_self(player, 1)
+    assert player.current_health == starting_health - 2
+
+    player.move_to_tokyo()
+    heal.heal_self(player, 1)
+    assert player.current_health == starting_health - 2
```

Comment:
Professor usually doesn't like two asserts in a single test as it makes for confusion, break these into two separate tests so when it fails, we know the condition that caused it to fail.

Reply:
Also make sure there is a newline at the end of the file